### ISSUE TITLE
Improve is couchbase up checking method

### ISSUE THREAD

**Issue by yurem** [GluuFederation/community-edition-setup#448]
Current method allows to check if Couchbase is up. But I suspect it not checking if all it services are up and ready.

Here is part from log:
```
19:32:17 07/03/18 Running: /sbin/service stop couchbase-server
19:32:17 07/03/18 Run: /sbin/service stop couchbase-server with result code: 1
19:32:17 07/03/18 Running: /sbin/service start couchbase-server
19:32:17 07/03/18 Run: /sbin/service start couchbase-server with result code: 1
19:32:17 07/03/18 Checking if couchbase was started. Try 1 ...
19:32:17 07/03/18 Couchbase was not started. Will retry after 5 seconds.
19:32:22 07/03/18 Checking if couchbase was started. Try 2 ...
19:32:22 07/03/18 Couchbase was not started. Will retry after 5 seconds.
19:32:27 07/03/18 Checking if couchbase was started. Try 3 ...
19:32:27 07/03/18 Running: /opt/couchbase/bin/couchbase-cli cluster-init --cluster localhost:28091 --cluster-username admin --cluster-password secret --services data,index,query,fts --cluster-ramsize 1024
19:32:28 07/03/18 SUCCESS: Cluster initialized
...
19:32:35 07/03/18 Running Couchbase query from file ./output/n1ql/index_gluu.n1ql
19:32:35 07/03/18 Running: /opt/couchbase/bin/cbq --user admin --password secret --engine localhost:28091 --file ./output/n1ql/index_gluu.n1ql
19:32:35 07/03/18 [31m ERROR 100 : N1QL: No query service found on this cluster [0m
...
Few imports errors
...
19:32:35 07/03/18 Importing ldif file ./output/attributes.ldif to Couchebase
19:32:35 07/03/18 Running Couchbase query from file ./output/n1ql/attributes.n1ql
19:32:35 07/03/18 Running: /opt/couchbase/bin/cbq --user admin --password secret --engine localhost:28091 --file ./output/n1ql/attributes.n1ql
19:32:37 07/03/18  Connected to : http://localhost:28091/. Type Ctrl-D or \QUIT to exit.

 Path to history file for the shell : /root/.cbq_history 
UPSERT INTO `gluu` (KEY, VALUE) VALUES ("attributes_@!4E71.8093.C25D.F8D5!0001!ECBB.7407!0005!29DA", {"dn": "inum=@!4E71.8093.C25D.F8D5!0001!ECBB.7407!0005!29DA,ou=attributes,o=@!4E71.8093.C25D.F8D5!0001!ECBB.7407,o=gluu", "gluuAttributeType": "string", "description": "XRI i-number, persistent non-reassignable identifier", "gluuStatus": "active", "gluuAttributeEditType": "admin", "urn": "urn:gluu:dir:attribute-def:inum", "oxAuthClaimName": "inum", "gluuAttributeOrigin": "gluuPerson", "objectClass": ["top", "gluuAttribute"], "gluuAttributeViewType": ["user", "admin"], "inum": "@!4E71.8093.C25D.F8D5!0001!ECBB.7407!0005!29DA", "displayName": "Inum", "gluuAttributeName": "inum"});
{
    "requestID": "b307bd1e-1af4-4f04-9f9f-11effb8e31e7",
    "signature": null,
    "results": [
    ],
    "status": "success",
    "metrics": {
        "elapsedTime": "83.721742ms",
        "executionTime": "83.699643ms",
        "resultCount": 0,
        "resultSize": 0,
        "mutationCount": 1
    }
}
...
successfully imported rest of files
```

**Comment by mbaser** [comment]
This function is added https://github.com/GluuFederation/community-edition-setup/blob/3e3799dbc0fda87ae314c6ac2e53c120e1e50bc3/setup.py#L3639
Please check if it works.

**Comment by yurem** [comment]
I've successfully installed 4.0.0 beta1 under CentOS 6.7.
But installation under Ubuntu 14.04 is not successfully. I've got same issue. Here is full setup log:
[setup.log](https://github.com/GluuFederation/community-edition-setup/files/2162193/setup.log)

**Comment by yurem** [comment]
I offer to run in loop with delay during 1 minute: 
```
/opt/couchbase/bin/cbq --user admin --password secret --engine localhost:28091 --file ./output/n1ql/test.n1ql
```
and check exit code. it should be 0 before we can load main N1QL scripts.

Sample output of `test.n1ql`:
```
SELECT 1 FROM `gluu`
```

**Comment by mbaser** [comment]
closed by commit 4ab7c1d
